src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -214,10 +214,32 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.clickEditSessionButton();
         editedSession.setInstructions(new Text("Made some changes"));
         feedbackEditPage.editFeedbackSession(editedSession.getEndTime(), editedSession.getStartTime(),
-                                        editedSession.getInstructions(), editedSession.getGracePeriod());
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), true);
 
         String expectedString = "The end time for this feedback session cannot be earlier than the start time.";
         feedbackEditPage.verifyStatus(expectedString);
+
+        ______TS("test discard changes to feedback session");
+
+        // Make changes to Feedback session
+        editedSession.setStartTime(Const.TIME_REPRESENTS_NOW);
+        editedSession.setEndTime(Const.TIME_REPRESENTS_LATER);
+        editedSession.setInstructions(new Text("Made some more changes"));
+        editedSession.setGracePeriod(15);
+        feedbackEditPage.editFeedbackSession(editedSession.getStartTime(), editedSession.getEndTime(),
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), false);
```

Comment:
UPDATE: If you notice, when you click `[Change]` alongside either of the Uncommon Settings Info Texts, and the FS is NOT in edit mode, it gets changed to edit mode. I am 95% certain that this is the cause of this unexpected behavior. **However, I cannot find the code that is responsible for this behaviour.**

All I found implementing behaviour of these buttons in `instructorFeedbackEdit.es6` were calls to some methods in its `document.ready` function imported from `common/instructorFeedbacks.es6`. The `.click` function for these buttons is implemented in the `bindUncommonSettingsEvents` function, which looks like this:
```JavaScript
function bindUncommonSettingsEvents() {
    $('#editUncommonSettingsSessionResponsesVisibleButton')
        .click(showUncommonPanelsForSessionResponsesVisible);
    $('#editUncommonSettingsSendEmailsButton')
        .click(showUncommonPanelsForSendEmails);
}
```

The two methods called in the above method look like this:
```JavaScript
function showUncommonPanelsForSessionResponsesVisible() {
    const $sessionResponsesVisiblePanel = $('#sessionResponsesVisiblePanel');

    $('#uncommonSettingsSessionResponsesVisible').after($sessionResponsesVisiblePanel);
    $sessionResponsesVisiblePanel.show();
    $('#uncommonSettingsSessionResponsesVisibleInfoText').parent().hide();
}

function showUncommonPanelsForSendEmails() {
    const $sendEmailsForPanel = $('#sendEmailsForPanel');

    $('#uncommonSettingsSendEmails').after($sendEmailsForPanel);
    $sendEmailsForPanel.show();
    $('#uncommonSettingsSendEmailsInfoText').parent().hide();
}
```

Nowhere in the above code blocks can I find how the FS gets put into Edit Mode if the button is clicked. It will be great if you could help with this @leeyimin.